analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -75,6 +77,8 @@ analyzer:
         - "Two-clause BSD-style license"
         declared_licenses_processed:
           spdx_expression: "BSD-2-Clause"
+          mapped:
```

Comment:
commit: Am I right that until now one design goal of the analyzer result (and scanner result as well) was `zero redundancy`? IIUC the data added by this change is redundant. Have you considered that?

I think it'd be valuable to explain this in the commit message - what do you think?

Reply:
I guess this goes into the same direction as my comment in the the chat, where I wrote

> Should we maybe only add one global list of mappings done in all packages to not further increase the output file size?

and @mnonnenmacher replied

> I think the impact on file size is minimal, because only declared licenses that are changed by the mapping are recorded. And storing the map globally would make the implementation a lot harder.

Reply:
I thought that the set of licenses which got actually mapped can be computed from the analyzer result already (without this change). And this part is probably redundant. But as you add the original strings and to what they actually got mapped it's not really redundant, so you're right @mnonnenmacher .